### PR TITLE
Improve Python 3 compatibility

### PR DESCRIPTION
**Pull request by galou**

### CODE REVIEW

File: kobuki_testsuite/src/kobuki_testsuite/drift_estimation.py [hunk 1]
```diff
@@ -160,8 +161,8 @@ def execute(self):
         with self.lock:
             if not self._initial_gyro_offset:
                 self._initial_gyro_offset = self._gyro_angle - self._scan_angle
-                print("Kobuki Testsuite: initial centre [%s]"%self._centred_gyro_angle)
-                print("Kobuki Testsuite: initial offset [%s]"%self._initial_gyro_offset)
+                print(("Kobuki Testsuite: initial centre [%s]" % self._centred_gyro_angle))
```
Comment (corot) [inline kobuki_testsuite/src/kobuki_testsuite/drift_estimation.py]: Unneeded double parentheses

File: kobuki_testsuite/src/kobuki_testsuite/motion_travel_forward.py [hunk 2]
```diff
@@ -81,7 +81,7 @@ def execute(self):
                 current_distance_sq = (self._current_pose.position.x - self._starting_pose.position.x)*(self._current_pose.position.x - self._starting_pose.position.x) + \
                                    (self._current_pose.position.y - self._starting_pose.position.y)*(self._current_pose.position.y - self._starting_pose.position.y)
                 #current_distance_sq += 0.01 # uncomment this and comment above for debugging
-                print("Distance %s"%math.sqrt(current_distance_sq))
+                print(("Distance %s"%math.sqrt(current_distance_sq)))
```
Comment (corot) [inline kobuki_testsuite/src/kobuki_testsuite/motion_travel_forward.py]: Double parentheses unneeded

**Review by corot — CHANGES_REQUESTED** [review]
Minor issues from automated formatter

File: kobuki_testsuite/src/kobuki_testsuite/motion_travel_forward.py (hunk 2 above)
Comment (galou) [reply kobuki_testsuite/src/kobuki_testsuite/motion_travel_forward.py]: Fixed!

**Review by corot — APPROVED** [review]

**Comment by corot** [comment]
Are you using this test suite?

**Comment by galou** [comment]
I don't use this test suite but the warning appeared when running `catkin build` and it looked that it could be fixed easily, so I just did it.

**Pull request merged**